src/Raven.Server/Documents/DocumentsChanges.cs

```diff
@@ -90,5 +97,20 @@ public void Disconnect(long id)
             if (Connections.TryRemove(id, out ChangesClientConnection connection))
                 connection.Dispose();
         }
+
+        public void LowMemory(LowMemorySeverity lowMemorySeverity)
+        {
+            if (lowMemorySeverity != LowMemorySeverity.ExtremelyLow)
+                return;
+
+            foreach (KeyValuePair<long, ChangesClientConnection> keyValue in Connections)
+            {
+                keyValue.Value.ClearSendQueue();
```

Comment:
this will be very confusing, suddenly changes api will stop working. In your case you had millions of objects, so maybe we should clear it if this exceeds some high threshold?

Reply:
Something to consider is if we shouldn't just _drop_ the connection.
That may be more robust since the user will get an error and be aware that they are missing stuff.

Reply:
done